README.md

```diff
@@ -1,9 +1,11 @@
 # Help desk
 
-Support for Infrastructure related issues for the Jenkins project
+Support for Infrastructure related issues for the Jenkins project.
 
-As of the 11th of January 2022, all existing Jira INFRA project have been imported in this repository as Github issues.
-See https://github.com/jenkins-infra/helpdesk/issues/9 for more details on the migration. 
+If you want to report an issue or a bug with the Jenkins website, wiki, or any other [services run by the Jenkins project](https://www.jenkins.io/projects/infrastructure/), [open an issue here](https://github.com/jenkins-infra/helpdesk/issues/new/choose)!
+
+As of the 11th of January 2022, all existing INFRA project issues have been migrated from [Jira](https://issues.jenkins.io/projects/INFRA/issues) to this repository as [Github issues](https://github.com/jenkins-infra/helpdesk/issues?q=label%3Aimported-jira-issue).
```

Comment:
is this relevant to people? it's very point in time information.

Better to send over in mailing lists than record it I would expect, how many people will want / need this information a week from now.

Reply:
Agreed on the "point in time", I've put this here for the moment, making it more visible for people clicking on links without reading them or their context, it could be removed next week or even now for what it worth.